src/components/base/HeroIcon.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <Icon
+    :icon="iconName"
+    :class="iconColor"
+  />
+</template>
+
+
+<script lang="ts">
+import { defineComponent, computed } from "vue";
+import { Icon } from "@iconify/vue";
```

Comment:
Please do not use `@iconify/vue`. 

Your import should look like this:

- https://github.com/launchbadge/kabuto-explorer/blob/master/src/components/EntityIcon.vue#L8